.github/workflows/go.yml

```diff
@@ -58,7 +58,7 @@ jobs:
             "runsc": {
               "path": "/usr/local/bin/runsc",
               "runtimeArgs": [
-                "--host-uds=all"
+                "--host-uds=create"
```

Comment:
If we stick with `DAC_OVERRIDE` when `Config.Rootless` is specified, the utility of the new `GRPCBrokerMultiplex` option is greatly reduced because the container would be able to write to host-created sockets without any additional changes. The only remaining benefit is this switch from `--host-uds=all` to `--host-uds=create`. I do think it's nice to know that it's impossible for host-created sockets to become visible and writable in the container, but it's less crucial than it was for the previous implementation.

I need to mull over the complexity vs benefit tradeoffs here a bit more before making a decision, but I'd welcome any other inputs on that.